data/variants.txt

```diff
@@ -1265,6 +1265,434 @@ ship "Heavy Shuttle" "Heavy Shuttle (Armed)"
 		"Chipmunk Plasma Thruster"
 		"Chipmunk Plasma Steering"
 		"Hyperdrive"
+		
+		
+ship "Kestrel" "KestrelEng (Particle)"
+	add attributes
+		"drag" -1.0
+		"fuel capacity" 100
+		"engine capacity" 30
+	outfits
+		"Outfits Expansion" 6
+		Hyperdrive
+		"Liquid Helium Cooler"
+		"A125 Atomic Steering"
+		"A865 Atomic Steering"
+		"A860 Atomic Thruster"
+		"Particle Cannon" 4
+		"Heavy Laser Turret" 4
+		"Liquid Nitrogen Cooler"
+		"Torpedo Launcher" 2
+		Torpedo 60
+		"Armageddon Core"
+		"S-970 Regenerator"
```

Comment:
This isn't a normally available outfit. It is unlocked at the end of the checkmate branch (and can be found on some marauders). (Likewise further down)

Reply:
You'll have to forgive me as I am confused. Are you referring to the s-970? Marauders already use these modules without any FW completion requirement.

Reply:
Happy to remove them, but that wouldn't be consistent with what is already in vanilla

Reply:
Marauders are a different case, we say that these are upgraded beam weapons that behave like electron beams, a different weapon entirely is probably a better solution to that problem but we don't have one of those.

Reply:
I understand now, thanks for clarification. I think the last commit fixes this.